jenkins_job_wrecker/modules/triggers.py

```diff
@@ -256,12 +256,65 @@ def ghprbtrigger(top, parent):
+        elif child.tag == 'blackListCommitAuthor':
+            ghpr['black-list-commit-author'] = child.text
+        elif child.tag == 'blackListLabels':
+            ghpr['black-list-labels'] = child.text.strip().split('\n')
+        elif child.tag == 'blackListTargetBranches':
+            ghpr['black-list-target-branches'] = [item[0].text for item in child]
+        elif child.tag == 'displayBuildErrorsOnDownstreamBuilds':
+            ghpr['display-build-errors-on-downstream-builds'] = True
+        elif child.tag == 'excludedRegions':
+            ghpr['excluded-regions'] = child.text.strip().split('\n')
+        elif child.tag == 'includedRegions':
+            ghpr['included-regions'] = child.text.strip().split('\n')
+        elif child.tag == 'skipBuildPhrase':
+            ghpr['skip-build-phrase'] = child.text
+        elif child.tag == 'whiteListLabels':
+            ghpr['white-list-labels'] = child.text.strip().split('\n')
         elif child.tag == 'whiteListTargetBranches':
             ghpr['white-list-target-branches'] = []
             for branch in child:
                 if branch[0].text is not None:
                     ghpr['white-list-target-branches'].append(branch[0].text.strip())
         elif child.tag == 'gitHubAuthId':
             ghpr['auth-id'] = child.text
+        elif child.tag == 'extensions':
+            extensions_prefix = "org.jenkinsci.plugins.ghprb.extensions."
+            for extension in child:
+                if extension.tag == extensions_prefix+"status.GhprbSimpleStatus":
+                    for attribute in extension:
+                        if attribute.tag == "commitStatusContext":
+                            ghpr['status-context'] = attribute.text
+                        elif attribute.tag == "triggeredStatus":
+                            ghpr['triggered-status'] = attribute.text
+                        elif attribute.tag == "startedStatus":
+                            ghpr['started-status'] = attribute.text
+                        elif attribute.tag == "statusUrl":
+                            ghpr['status-url'] = attribute.text
+                        elif attribute.tag == "addTestResults":
+                            ghpr['status-add-test-results'] = attribute.text
+                        elif attribute.tag == "completedStatus":
+                            for status in attribute:
+                                if status[1].text == "SUCCESS":
+                                    ghpr['success-status'] = status[0].text
+                                elif status[1].text == "FAILURE":
+                                    ghpr['failure-status'] = status[0].text
+                                elif status[1].text == "ERROR":
+                                    ghpr['error-status'] = status[0].text
+                elif extension.tag == extensions_prefix+"comments.GhprbBuildStatus":
+                    for attribute in extension[0]:  # messages
```

Comment:
a proper implementation would be to iterate over the children of `extension`, check the tag of the child to match for `messages` otherwise raise NotImplementedError